.github/workflows/autopilot-branch-to-latest.yaml

```diff
@@ -0,0 +1,254 @@
+name: Autopilot - Branch Versions to Latest
```

Comment:
We have a similar check here, it probably should be removed
https://github.com/k0sproject/k0s/blob/main/.github/workflows/go.yml#L241-L260

Reply:
This workflow is going to be manual for a while to ensure that the wheels don't fall off, so it would be good to keep the current autopilot integration upgrade tests in place.

Once we see the wheels are staying on, we can move this nightly.